src/ssb_project_cli/ssb_project/app.py

```diff
@@ -356,11 +458,19 @@ def create_github(
 
 
 def projectname_from_currfolder(curr_path: Optional[str]) -> str:
+    """Retrives project name from poetry's toml-config in cwd.
+
+    Args:
+        curr_path: Optional string of current path.
+
+    Returns:
+        str: Project name from poetry`s toml-config
+    """
     # Record for reset later
     curr_dir = os.getcwd()
     # Find root of project, and get projectname from poetry's toml-config
     while "pyproject.toml" not in os.listdir():
-        os.chdir("../")
+        os.chdir("../../dapla_hurtigstart_cli/")
```

Comment:
This doesn't look right, that name shouldn't exist anymore

Reply:
Reverted to projectname_from_currfolder from main.